scheduler/scheduler.cpp

```diff
@@ -2245,6 +2269,11 @@ int main(int argc, char *argv[])
                 max_fd--;
             }
         }
+
+        for (list<CompileServer *>::const_iterator it = cs_in_tsts.begin();
+                it != cs_in_tsts.end(); ++it) {
+            (*it)->inConnectionResponse(max_fd, read_set, write_set);
```

Comment:
We should be checking if the fd is set in either read_set or write_set here.  What we do about it being set belongs to compileServer, but that it is set belongs here.